src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -2874,6 +2874,10 @@ void PolicyHandler::UpdateHMILevel(ApplicationSharedPtr app,
 bool PolicyHandler::CheckModule(const PTString& app_id,
                                 const PTString& module) {
   const auto policy_manager = LoadPolicyManager();
+  if (!PolicyEnabled()) {
```

Comment:
@RomanReznichenkoLuxoft please move this check above line 2876

Reply:
done